app/assets/javascripts/discourse/tests/acceptance/topic-edit-timer-test.js

```diff
@@ -354,7 +354,6 @@ acceptance("Topic - Edit timer", function (needs) {
 
     const expected = [
       I18n.t("time_shortcut.tomorrow"),
-      I18n.t("time_shortcut.this_weekend"),
```

Comment:
This would fail because `suggest_weekends_in_date_pickers` is not set? I'd suggest adding the setting and keep the test as is, given that `suggest_weekends_in_date_pickers=true` by default.

Reply:
Makes sense, corrected the tests.